bootstrap/dev/venv.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh -xe
 # Developer virtualenv setup for Let's Encrypt client
 
+export VENV_ARGS="--python python2"
```

Comment:
That breaks (at least) Debian squeeze as per http://letsencrypt.readthedocs.org/en/latest/using.html#debian,  but meeeh...